BoxContentSDK/BoxContentSDK/Operations/BOXAPIDataOperation.m

```diff
@@ -294,6 +294,17 @@ - (void)progressWithTotalBytesWritten:(int64_t)totalBytesWritten totalBytesExpec
     }
 }
 
+- (void)didFinishDownloadingToURL:(NSURL *)location
+{
+    if (self.destinationPath != nil) {
+        NSError *error = nil;
+        [[NSFileManager defaultManager] moveItemAtPath:location.path toPath:self.destinationPath error:&error];
+        if (self.error == nil) {
```

Comment:
Is there a case where `self.error != nil`, and this gets called? I guess I'm fine with this check, anyway.